packages/schematics/angular/utility/latest-versions/package.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
NB: this is a seperate child directory due avoid ESLint errors from the `extraneous-dependencies` rule.